run_ci.sh

```diff
@@ -105,7 +105,7 @@ if [ -d "/home/notstack/ironic-image" ] ; then
     export UPSTREAM_IRONIC=true
 fi
 # Starting from Openshift 4.9 the ironic-inspector container is not used anymore
-if is_lower_version $OPENSHIFT_VERSION 4.9 && [ -d "/home/notstack/ironic-inspector-image" ] ; then
+if is_lower_version $(openshift_version) 4.9 && [ -d "/home/notstack/ironic-inspector-image" ] ; then
```

Comment:
I don't think this file is used at all anymore, so we could look at removing it in a followup.

Reply:
#1352